quantum/quantum.c

```diff
@@ -451,7 +447,7 @@ bool process_record_quantum(keyrecord_t *record) {
         return false;
     }
 
-    return process_action_kb(record);
+    return true;
```

Comment:
The previous if statement would be a lot cleaner if it just returned a value.  `if (!(` could be just `return (`.  Right now it effectively says `if not stuff, return false, else return true` when it could just say `return stuff`.